reports/Treat2015Visulize.Rmd

```diff
@@ -0,0 +1,262 @@
+---
+title: "Treat peatlands 2015"
+author: "K Todd-Brown (ktoddbrown@gmail.com)"
+date: "8/3/2017"
+output:
+  pdf_document:
+    fig_caption: yes
+    toc: yes
+  html_document: default
+---
+
+```{r setup}
+library(SoilDataR) #library(devtools); install_github("ktoddbrown/soilDataR")
+library(ggplot2) #make pretty plots
+library(plyr) #data management
+library(dplyr)
+library(knitr)
+library(lubridate)
+library(tidyr)
+
+#mapping librarys to help with global/regional plots
+library(ggmap)
+library(maps)
+library(mapdata)
+library(fiftystater)
+```
+
+```{r loadData, cache=TRUE}
+Treat <- SoilDataR::processData_Treat2015(dir='../repoData/Treat_2015/')
+#ISCN3 <- SoilDataR::processData_ISCN3(dir='../repoData/ISCN_3/')
+```
+
+#Observation time
+```{r obsTime}
+location.df <- unique(Treat$field[, c('lat', 'long', 'observation_date', 
+                                       'state', 'country')])
+location.df$year <- location.df$observation_date
+
+location.df$obsDate <- ymd(paste(location.df$year, 1, 1, sep="-"))
+location.df <- location.df[order(location.df$lat, location.df$long, location.df$obsDate),]
+location.df$yrCut <- cut(location.df$year, c(1950, 1960, 1970, 1980, 1990, 2000, 2013), dig.lab=4)
```

Comment:
See comment above–what do these years mean?

Reply:
Reworked so that this is in 10yr increments from the start to the end of the time line f60dc4724d7956b2186f7ae06be14767f39d77d2